lib/mongoid_rails_migrations/active_record_ext/migrations.rb

```diff
@@ -216,7 +236,10 @@ def rollback(migrations_path, steps=1)
 
       def rollback_to(migrations_path, target_version)
         all_versions = get_all_versions
-        rollback_to = all_versions.index(target_version.to_i) + 1
+        target_version_index = all_versions.index(target_version.to_i)
+        raise UnknownMigrationVersionError.new(target_version) if target_version_index.nil?
```

Comment:
This is fixing a previous bug, when a wrong version was used in:
`rails db:rollbkack_to VERSION=fake_version`
this code was raising the following error:
```
rails aborted!
NoMethodError: undefined method `+' for nil:NilClass
/home/seb/projects/mongoid_rails_migrations/lib/mongoid_rails_migrations/active_record_ext/migrations.rb:239:in `rollback_to'
/home/seb/projects/mongoid_rails_migrations/lib/mongoid_rails_migrations/mongoid_ext/railties/database.rake:113:in `block (3 levels) in <main>'
/home/seb/projects/mongoid_rails_migrations/lib/mongoid_rails_migrations/active_record_ext/migrations.rb:294:in `with_mongoid_client'
/home/seb/projects/mongoid_rails_migrations/lib/mongoid_rails_migrations/mongoid_ext/railties/database.rake:112:in `block (2 levels) in <main>'
/home/seb/ringcentral/engage-digital/bin/rails:8:in `<top (required)>'
/home/seb/ringcentral/engage-digital/bin/spring:16:in `<top (required)>'
bin/rails:3:in `load'
bin/rails:3:in `<main>'
Tasks: TOP => db:rollback_to
(See full trace by running task with --trace)
```